src/rasberry_perception/detection/interfaces/gazebo_berries.py

```diff
@@ -53,13 +79,31 @@ def get_detector_results(self, request):
         detections = Detections()
 
         try:
-            image = ros_numpy.numpify(request.image)
-            if request.image.encoding == "rgb8":
-                image = image[..., ::-1]
-            predictions = self.predictor(image)
-            # Get the bounding boxes/detections for the image here
+            rois  = []
+            poses = []
+            for pose in self.obj_poses:
+                transform = self.tf_buffer.lookup_transform(self.ref_frame, 'odom', rospy.Time())
```

Comment:
Did you mean to remove this?

Reply:
No, this transforms to reference frame of camera.